modules/hcp-ts-hc-stuck.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ts-hc-stuck_{context}"]
+= HostedCluster resource is stuck in a partial state
+
+If a hosted control plane is not coming fully online because a `HostedCluster` resource is pending, identify the problem by checking prerequisites, resource conditions, and node and operator status.
```

Comment:
```suggestion
If a hosted control plane is not coming fully online because a `HostedCluster` resource is pending, identify the problem by checking prerequisites, resource conditions, and node and Operator status.
```
Even though it does not specify a specific Operator, should it be init-capped?

Reply:
Good catch. That's a stylistic difference between the RHACM and OCP docs. I'll capitalize "Operator" to align with the OCP docs.